src/main/java/com/redhat/devtools/intellij/tektoncd/tkn/TknCli.java

```diff
@@ -79,6 +78,26 @@ public boolean isTektonAware(KubernetesClient client) {
         }
     }
 
+    @Override
+    public List<String> getPipelines(String namespace) throws IOException {
+        String output = ExecHelper.execute(command, "pipeline", "ls", "-n", namespace, "-o", "jsonpath={.items[*].metadata.name}");
+        return Arrays.stream(output.split("\\s+")).filter(item -> !item.isEmpty()).collect(Collectors.toList());
+    }
+
+    @Override
+    public List<PipelineRun> getPipelineRuns(String namespace, String pipeline) throws IOException {
+        String json = ExecHelper.execute(command, "pipelinerun", "ls", pipeline, "-n", namespace, "-o", "json");
+        return JSON_MAPPER.
+                registerModule(new SimpleModule().addDeserializer(List.class, new PipelineRunDeserializer())).
```

Comment:
I'm wondering if registering a new module for each call does not lead to memory leaks

Reply:
I saw you already created a PR on fabric8 repo to solve the issue you found and they seem to answer fastly. For the moment i came back to the static block so we should be safe with any possible memory leaks. When the issue on fabric8 will be solved we can move to that one